packages/api-providers/src/contracts/wrappers/webb-vanchor.ts

```diff
@@ -490,7 +490,8 @@ export class VAnchorContract {
 
       const inputNote = await Note.generateNote(noteInput);
       inputNotes.push(inputNote);
-      inputIndices.push(inputUtxo.index);
+      // TODO: @nepoche ensure index is defined
```

Comment:
CircomUtxo will always return a '0' for the index if it isn't supplied during Utxo generation.